### DISCUSSION TITLE
General syntax question, on SQL query SELECT Statement (SQL Quiz)

### DISCUSSION THREAD

**Discussion by mynugraErlangga** [teamalgoritma/community#652] [Q&A] [answered]
Selamat Siang

Background: Saat mengerjakan quiz untuk SQL Query module, saya disediakan skeleton code line untuk menjawab pertanyaan as follows:
![image](https://github.com/teamalgoritma/community/assets/125937571/86dd83b5-78ff-4880-9f26-315cc33b7b7a)

Code: 
customer_total = pd.read_sql_query(
    '''
    SELECT SUM(invoices.Total) as Total, 
    customers.FirstName || ' ' || customers.LastName AS CustomerName  
   and so on)

My specific question: 
Menggabungkan columns FirstName dan LastName perlu  || '  ' || ,   APAkah bisa  hanya   ||  ??

Terima kasih jawabannya

Note that I Have found out the answer for that specific question-number with no problems

**Comment by idoali** [comment, marked as answer]
Terima kasih pak atas pertanyaannya

Jawabannya tidak bisa pak. Tanda `|| ' ' ||` memang harus digunakan jika kita ingin menggabungkan string dari dua kolom dengan spasi. Kita tidak bisa menggunakan `||` saja. 

Untuk penjelasan lengkapnya, bapak bisa membuka dokumentasi ini https://learnsql.com/cookbook/how-to-concatenate-strings-in-sql/ 

Semoga cukup menjawab pak
